web/libs/editor/src/components/Comments/Comment/CommentItem.scss

```diff
@@ -151,4 +151,9 @@
     justify-content: space-between;
     padding: var(--comment-item-padding);
   }
+
+  &__classifications {
+    font-size: 13px;
+    letter-spacing: .05em;
```

Comment:
That is a very specific sizing, is it something pulled from the design file? I haven't seen us ever use `em` or a font size of 13px.

Reply:
I was emulating the look of these typography options in figma; know that it's a bit weird. Changing from em to px

![image](https://github.com/user-attachments/assets/fcdd8c92-2bf8-4450-aafb-cc717c0a8dd7)

Reply:
(feel free to reopen btw)